docs/Style-Guide.md

```diff
@@ -0,0 +1,180 @@
+
+More Like this: To register a parametric path, put a colon before the parameter name. Using colon lets the framework know it is a parametric path and not a wild card.
+
+### Avoid adding video or image content 
+
+
+Do not add videos or screenshots in the documentation. It is easier to keep under version control. Videos and images will eventually end up becoming outdated as new updates keep developing. Instead, make a referral link or a youtube video. You can add links by using `[Title](www.websitename.com)` in the markdown.
+
+**Example**
+
+```
+To learn more about hooks, See [Fastify hooks](https://www.fastify.io/docs/latest/Hooks).
+```
+
+Result:
+>To learn more about hooks, See [Fastify hooks](https://www.fastify.io/docs/latest/Hooks/).
+
+
+
+### Avoid plagiarism
+
+Make sure you evade copying other people's work, keep it as original as possible. You can learn from what they've done and also reference where it's from if you used a particular quote from their work.
+
+
+## Word Choice
+
+
+There are a few things you need to use and avoid when writing your documentation which will improve readability for readers and also make documentations neat, direct and clean all through your writing process.
+
+
+### When to use the second person "you" as the pronoun
+
+When writing articles or step-by-step guides, your content should communicate directly to readers who is the second person ("you") addressed. It is easier to give them direct instruction on what to do on a particular topic. To see an example, visit the [Plugins-guide.md](https://github.com/fastify/fastify/blob/df9fbc183fa05fa1a23781b3f11fbf81f6854033/docs/Plugins-Guide.md) page on Github. 
+
+**Example**
+
+Less Like this: we can use the following plugins.
+
+More like this: You can use the following plugins.
+
+> According to [Wikipedia](#), ***You*** is usually a second person pronoun. Also, used to refer to an indeterminate person, as a more common alternative to a very formal indefinite pronoun.
+
+## When to avoid second person "you" as the pronoun
+
+One of the main rules of formal writing such as references, synthesis and also in API documentation is to avoid second person ("you"). It should not directly address the user even when the word "you" is almost unavoidable when writing, but there are ways to avoid second person. Below are some alternative examples you can use to evade them. 
+
+**Example**
+
+Less Like this: You can use the following recommendation as an example.
+
+More like this: As an example, The following are recommendations to use.
+
+To view a documented example, visit the [Decorators.md](https://github.com/fastify/fastify/blob/df9fbc183fa05fa1a23781b3f11fbf81f6854033/docs/Decorators.md) page on Github. 
+
+
+### Avoid writing condescending languages.
```

Comment:
```suggestion
### Avoid using condescending terms
```

Reply:
FWIW, I don't consider any of these "condescending." What they are is poor form. They are the usage of common colloquialisms, which are easily depended upon within the informal style promoted by the "guides style." So we are making contradictory recommendations by 1. suggesting guides can be written in an informal style and 2. prohibiting informal language.

But, whatever.

Reply:
Thank you for pointing that out.

I do appreciate you taking the time to go through this. Taking your highlighted concern into consideration, it's quite understandable those words are regarded as poor forms. According to [this article](https://dev.to/meeshkan/how-to-remove-condescending-language-from-documentation-4a5p) recommended by @delvedor, "condescending" _was defined as words that are offensive and also words that give an impression of something that makes people feel inadequate or hurt if they don't immediately understand it_.

The goal was to follow that same approach, I could make that topic more comprehensive to avoid context that may seem contradictory with the style guide.

Still, I'd really appreciate any feedback or recommendations that can help me improve my current work. 🙂